src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,23 @@
+%animation-settings {
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+  transform-origin: bottom center;
+  animation-name: move;
```

Comment:
The animation name 'move' is too generic. Consider using a more descriptive name such as 'rotate-clockwise'.